landoui/landoapi.py

```diff
@@ -69,7 +69,14 @@ def request(self, method, url_path, *, require_auth0=False, **kwargs):
                 },
             )
 
-            data = response.json()
+            try:
```

Comment:
Anything we can do here to reduce nesting? This is a try within a try.

Reply:
I think we want to try and decode the JSON and return it before raising the requests exception. I suppose I could add this to a function?